src/test/java/seedu/address/model/person/MeetingTest.java

```diff
@@ -44,7 +46,7 @@ public void isValidStartAndEndTime() {
     }
 
     @Test
-    public void isOverlap() {
+    public void isOverlap() throws CommandException {
```

Comment:
Also, as naming convention goes for test cases, the name for this test should be `test_isOverlap` instead of `isOverlap`, rationale being to help other devs figure out which `isOverlap` method is the legit method and which one is the test method quicker. Please also apply this to all the other test cases you wrote (i.e. isValidStartAndEndTime, equals, isValidLocation) so that they also follow the naming convention

Reply:
The code checks were the ones that prompted me to add the descriptor on the methods. After catching and resolving the exception within the methods, it should all be okay now.. as long as I handled it fine. Do give me furhter updates in case something else needs to change

I will change the naming convention